diff --git a/closure/private/defs.bzl b/closure/private/defs.bzl
index 286057e..fabb3c7 100644
--- closure/private/defs.bzl
+++ closure/private/defs.bzl
@@ -213,7 +213,7 @@ def find_js_module_roots(srcs, workspace_name, label, includes):
                         break
                 if not found:
                     fail("No srcs found beginning with '%s/'" % include)
-            for root in roots.to_list():
+            for root in roots:
                 magic_roots.append("%s/%s" % (root, prefix))
         roots += magic_roots
     return depset(roots)
